Cargo.toml

```diff
@@ -93,3 +97,9 @@ lto = "thin"
 [profile.release-debug-info]
 inherits = "release"
 debug = true
+
+[patch.crates-io]
+pipewire = { git = "https://github.com/bim9262/pipewire-rs.git", branch = "builder_array" }
+pipewire-sys = { git = "https://github.com/bim9262/pipewire-rs.git", branch = "builder_array"}
+libspa = { git = "https://github.com/bim9262/pipewire-rs.git", branch = "builder_array" }
+libspa-sys = { git = "https://github.com/bim9262/pipewire-rs.git", branch = "builder_array" }
```

Comment:
Do you plan to upstream these changes?

Reply:
I do. I have opened an issue as I can't create a fork (permission issue) on freedesktop's gitlab instance